site/docs/master/website-guidelines.md

```diff
@@ -0,0 +1,38 @@
+# Website Guidelines
+
+## Running the website locally
+
+When making changes to the website, please run the site locally before submitting a PR and manually verify your changes.
+
+Under the `velero/site/` directory, run:
+
+```bash
```

Comment:
There's also the `make serve-docs` target.

Reply:
Ah... great. Adding it.